src/main/java/net/greatstart/controllers/ProjectController.java

```diff
@@ -25,15 +33,25 @@
 public class ProjectController {
     private static final String REDIRECT_TO_PROJECTS = "redirect:/project/";
     private static final String PROJECTS = "project/projects";
-    private static final String PROJECT = "project";
+    private static final String PROJECT = "dtoProject";
+
+    private final Logger logger = LoggerFactory.getLogger(ProjectController.class);
 
     private ProjectService projectService;
     private UserService userService;
+    private CategoryService categoryService;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "categoryService" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)